lightsaber/src/main/java/schwarz/it/lightsaber/LightsaberDaggerProcessor.kt

```diff
@@ -79,27 +75,17 @@ public class LightsaberBindingGraphPlugin : BindingGraphPlugin {
             "Lightsaber.CheckUnusedModules",
         )
     }
-
-    private fun Issue.getMessage(): String {
-        return "$codePosition: $message [$rule]"
-    }
 }
 
-private fun runRule(check: Boolean, ruleName: String, rule: () -> List<Finding>): List<Issue> {
+fun runRule(check: Boolean, ruleName: String, rule: () -> List<Finding>): List<Issue> {
```

Comment:
```suggestion
private fun runRule(check: Boolean, ruleName: String, rule: () -> List<Finding>): List<Issue> {
```

I think that we don't need this change.